src/i18n/index.js

```diff
@@ -0,0 +1,70 @@
+// @flow
+import en from '../translations/en.json';
+import it from '../translations/it.json';
```

Comment:
Can we try reading files from directory? 
e.g.
```
const files = fs.readdirSync('../translations');
const translations = files.reduce((dict, file) => 
   const locale = file.split('.').shift(); // en | it | en_US
   dict[locale] = require(`../translations/${file}`;
   return dict;
), {});
```

Reply:
In this way we can manage localization by adding or removing files from `translations` directory